premake-core.code-workspace

```diff
@@ -0,0 +1,61 @@
+{
+	"folders": [
+		{
+			"name": "root",
+			"path": ".",
+		}
+	],
+	"settings": {
+		"files.exclude": {
+			"build": true,
+			"bin": true,
+			"MyLocation": true,
+		}
+	},
+	"tasks": {
+		"version": "2.0.0",
+		"tasks": [
+			{
+				"label": "bootstrap",
+				"type": "shell",
+				"command": "call Bootstrap.bat"
+			},
+		]
+	},
+	"launch": {
+		"version": "0.2.0",
+		"configurations": [
+			{
+				"name": "test",
+				"type": "lua-local",
+				"request": "launch",
+				"cwd": "${workspaceFolder}",
+				"program": {
+					"command": "bin\\release\\premake5.exe",
```

Comment:
'.exe' (and 'call xx.bat'), so it assumes windows, whereas vscode is cross platform.
Not sure how vscode handles platform settings...

Reply:
Thanks for pointing that out.
I will remove the exe at the end.

The bootstrap task uses [platform-specific settings](https://code.visualstudio.com/docs/editor/debugging#_platformspecific-properties) to work on multiple platforms